test/_external/test262/blacklist

```diff
@@ -0,0 +1,101 @@
+# export * from ns
```

Comment:
Could you rename this list to `skiplist`

Reply:
done.